src/main/java/pl/Lukasz/Calories/MvcConfig.java

```diff
@@ -0,0 +1,17 @@
+package pl.Lukasz.Calories;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class MvcConfig implements WebMvcConfigurer {
+
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addViewController("/home").setViewName("home");
+        registry.addViewController("/").setViewName("home");
+        registry.addViewController("/hello").setViewName("hello");
+        registry.addViewController("/login").setViewName("login");
```

Comment:
Nie musiałeś robić widoków (HTML) - mogło zostać na samych REST'ach - ale jak chcesz generować to spoko ;)